static/js/src/advantage/error-handler.js

```diff
@@ -1,225 +0,0 @@
-];
-
-const INCORRECT_NUMBER_CODES = [
-  "incorrect_number",
-  "invalid_number",
-  "incomplete_number",
-];
-
-const INCORRECT_ZIP_CODES = ["incorrect_zip", "incomplete_zip"];
-
-const PAYMENT_ERRORS = [
-  "invoice payment failed",
-  "payment method error",
-  "invalid customer information",
-];
-
-function customErrorResponse(errorData) {
-  const code = errorData.code;
-  let error = {
-    message: errorData.message,
-    type: "notification",
-  };
-
-  if (
-    INSUFFICIENT_FUNDS_CODES.includes(code) ||
-    error.message.includes("Your card has insufficient funds")
-  ) {
-    error.message =
-      "That card doesn’t have enough funds to make this payment. Please contact your card issuer, or try a different card.";
-  } else if (INCORRECT_NUMBER_CODES.includes(code)) {
-    error.message =
-      "That card number is incorrect. Check the number and try again.";
-    error.type = "card";
-  } else if (INCORRECT_CVC_CODES.includes(code)) {
-    error.message =
-      "That CVC number is incorrect. Check the number and try again.";
-    error.type = "card";
-  } else if (INCORRECT_ZIP_CODES.includes(code)) {
-    error.message =
-      "That ZIP/postal code is incorrect. Check the code and try again.";
-    error.type = "card";
-  } else if (INCORRECT_EXPIRY_CODES.includes(code)) {
-    error.message =
-      "That expiry date is incorrect. Check the date and try again.";
-    error.type = "card";
-  } else if (code === "tax_id_invalid") {
-    error.message =
-      "That VAT number is invalid. Check the number and try again.";
-    error.type = "vat";
-  } else if (code === "card_not_supported") {
-    error.message =
-      "That card doesn’t allow this kind of payment. Please contact your card issuer, or try a different card.";
-    error.type = "notification";
-  } else if (code === "currency_not_supported") {
-    error.message =
-      "That card doesn’t allow payments in this currency. Please contact your card issuer, or try a different card.";
-    error.type = "notification";
-  } else if (code === "expired_card") {
-    error.message = "That card has expired. Try a different card.";
-    error.type = "notification";
-  } else if (code === "testmode_decline") {
-    error.message = "Test completed.";
-    error.type = "dialog";
-  } else if (CARD_DECLINED_CODES.includes(code)) {
-    error.message =
-      "Your card has been declined. Please contact your card issuer, or try a different card.";
-    error.type = "notification";
-  }
-
-  return error;
-}
-
-function errorString(message) {
-  const unexpectedErrorStrings = [
-    "unexpected error setting customer payment method: ",
-    "unexpected error updating customer information: ",
-    "unexpected error creating customer: ",
-  ];
-  let error = false;
-
-  unexpectedErrorStrings.forEach((string) => {
-    if (message.includes(string)) {
-      error = string;
-    }
-  });
-
-  return error;
-}
-
-export function parseForErrorObject(data) {
```

Comment:
These functions weren't used. The one that is used (`getErrorMessage`) has been moved to `translateErrors.tsx`